### PR TITLE
Landing page styling

### PR DESCRIPTION
**Pull request by bbrady00**
Fixed date function for holiday API (won't show anything at the moment because no public holiday )
Added toast for successful expense log
Add Weather Icon to Locale Storage

### CODE REVIEW

File: assets/css/landing.css [hunk 1]
```diff
@@ -1,7 +1,90 @@
...
+  border-radius: 50px;
 }
 
 #value-input {
-    max-width: 18vw;
-}
\ No newline at end of file
+  max-width: 18vw;
+}
+
```
Comment (bbrady00) [inline assets/css/landing.css:8]: styling for landing page

File: assets/css/landing.css [hunk 2]
```diff
@@ -1,7 +1,90 @@
...
+/* Enhancements for the footer */
+.footer {
+  padding: 2em 1em;
+  background-color: #ffdd57;
+  color: #4a4a4a;
+  font-size: 1.1rem;
+}
+
```
Comment (bbrady00) [inline assets/css/landing.css:59]: Add pop up for successful expense log

File: assets/js/landing.js [hunk 3]
```diff
@@ -1,184 +1,215 @@
...
-const wind = document.getElementById("wind-data");
-const humidity = document.getElementById("humidity-data");
-const condition = document.getElementById("conditions");
-const rainChance = document.getElementById("rain-data");
 const holidayElement = document.getElementById("holiday");
 const dateEl = document.getElementById("date-header");
+
 const submitBtn = document.getElementById("saveBtn");
 const expenseInputEl = document.getElementById("expense-select");
 const amountInputEl = document.getElementById("value-input");
```
Comment (bbrady00) [inline assets/js/landing.js:7]: create function to format date

File: index.html [hunk 4]
```diff
@@ -85,6 +85,7 @@ <h3 class="title is-3 has-text-grey-darker">Chance of Rain</h3>
             <div class="my-2">
               <h3 class="title is-3 has-text-grey-darker">Condition</h3>
               <div class="weather-icon title is-4 has-text-link" id="conditions"></div>
+              <img id="weather-icon" src="" alt="Weather Icon">
```
Comment (bbrady00) [inline index.html:61]: add weather img placeholder

**Review by MelodyYan96 — APPROVED** [review]

**Pull request merged**